### PR TITLE
Feature wishlist

### LINKED ISSUE #7: Add Wishlist Management
- User must be able to add and remove product from wishlist.
- Add a button to each component to move the products to cart

### PR DESCRIPTION
**Pull request by OmKakatkar**
## Adds Save and Remove from  Wishlist Feature
- Adds some styles for loader in Card
- Adds a api constants file with all api endpoints currently being used.
- User can add or remove the item from wishlist in Product listing page.
- User can remove the items from wishlist in Wishlist page
-  Current tally of wishlist items is reflect on the navbar

image
image
image

### CODE REVIEW

File: src/components/EcommerceCard.js [hunk 1]
```diff
@@ -1,28 +1,49 @@
+import { useState } from 'react';
 import { faHeart } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { useAuth } from '../context/auth-context';
+import { addToWishList, removeFromWishList } from '../util/product-request';
 import Card from './Card';
 import './EcommerceCard.css';
+import { useProduct } from '../context/product-context';
+import loader from '../assets/loaders/loader.gif';
```
Comment (VishakAmin) [inline src/components/EcommerceCard.js:10]: Separate the module import with local import will increase the readability
Ex:
```
//All module imports at the start
import axios from "axios";

//All local import
import Card from './Card';
```

**Review by VishakAmin — COMMENTED** [review]
Looks good left a comment please have a look

**Review by khushijohri2001 — APPROVED** [review]
Clean code!!

File: src/components/EcommerceCard.js (hunk 1 above)
Comment (OmKakatkar) [reply src/components/EcommerceCard.js:10]: Have updated the code to make it more readable by separating the imports as you mentioned. Thanks for pointing it out.

**Pull request merged**